fares-data-build-tool/fdbt-site
# fix: retain user-entered multiProduct info

# Description

-   This PR refactors the MULTIPLE_PRODUCT_ATTRIBUTE so that the information the user has entered on the /multipleProducts page is retained, even when they have progressed further through the journey.
 
# Testing instructions

-   Pull down this branch and run the site locally. Navigate through a multi product period ticket journey (either of geoZone or multiServices) and check that the multipleProducts, multipleProductValidity and ticketConfirmation pages behave as they did previously.
-   Once you have navigated through the journey to the ticketConfirmation page, click one of the 'Change' links that will take you back to the /multipleProducts page and check that the info you previously entered is rendered on the page.

# Type of change


# Checklist:
